src/create-file/index.js

```diff
@@ -0,0 +1,62 @@
+import fsPromises from 'fs/promises';
+import { initialize, log } from '../workerTemplate.js';
+const workerQueue = process.env.WORKERQUEUE;
+const resultQueue = process.env.RESULTSQUEUE;
+const rabbitHost = process.env.RABBITHOST;
+const rabbitUser = process.env.RABBITUSER;
+const rabbitPass = process.env.RABBITPASS;
+
+/**
+ * Creates a file at the given destination path and with the given content.
+ * @param {Object} workerJob The job object
+ * @param {Array} inputs The inputs for this process
+ *   First input {String|Buffer|TypedArray|DataView} is the content of the file to be written
+ *   Second input {String} is the absolute path to store the content to. If filename and suffix
+ *     are ommited, they will be generated (file.txt). Full path is returned as output argument.
+ */
+const callbackWorker = async (workerJob, inputs) => {
+    const content = inputs[0];
+    const destinationFileName = inputs[1];
+    const hasSuffix = destinationFileName.includes(".") &&
```

Comment:
checking with a build-in function my be more robust. for example with https://nodejs.org/dist/latest-v18.x/docs/api/path.html#pathextnamepath